frontend/src/pages/Policy/policyContent/HowToUseTheDataContent.tsx

```diff
@@ -0,0 +1,209 @@
+import { METRIC_CONFIG } from '../../../data/config/MetricConfig'
+import { RACE } from '../../../data/utils/Constants'
+import { Fips } from '../../../data/utils/Fips'
+import Custom100kBarChart from '../../../reports/Custom100kBarChart'
+import CustomBreakdownSummary from '../../../reports/CustomBreakdownSummary'
+import CustomChoroplethMap from '../../../reports/CustomChoroplethMap'
+import CustomDisparityBarChart from '../../../reports/CustomDisparityBarChart'
+import CustomRateTrendsLineChart from '../../../reports/CustomRateTrendsLineChart'
+import CustomShareTrendsLineChart from '../../../reports/CustomShareTrendsLineChart'
+import CustomUnknownMap from '../../../reports/CustomUnknownMap'
+
+export const dataVisuals = [
+	{
+		title: 'Rate Choropleth Map',
+		description: 'The rate map shows the geographic distribution of gun violence across the United States. Choropleth maps show the racial/ethnic group with the highest disparity.',
+        sectionId: '#rate-choropleth-map',
+		details: {
+			demographicGranularities: ['Race/ethnicity', 'sex', 'age'],
+			geographicLevels: ['National', 'states', 'counties', 'territories'],
```

Comment:
Let's create an array at the top of this page `const gunViolenceGeographicLevels = ['National', 'states']` and then use that throughout since it's the same for every card